profile/config.go

```diff
@@ -119,6 +137,39 @@ var securityLevelSettings = []string{
 	CfgOptionDisableAutoPermitKey,
 }
 
+var (
+	// SPNRulesQuickSettings is a list of countries the SPN currently is present in
+	// as quick settings in order to help users with SPN related policy settings.
+	// This is a quick win to make the MVP easier to use, but will be replaced by
+	// a better solution in the future.
+	SPNRulesQuickSettings = []config.QuickSetting{
+		{Name: "Exclude Canada (CA)", Action: config.QuickMergeTop, Value: []string{"- CA"}},
+		{Name: "Exclude Finland (FI)", Action: config.QuickMergeTop, Value: []string{"- FI"}},
+		{Name: "Exclude France (FR)", Action: config.QuickMergeTop, Value: []string{"- FR"}},
+		{Name: "Exclude Germany (DE)", Action: config.QuickMergeTop, Value: []string{"- DE"}},
+		{Name: "Exclude Israel (IL)", Action: config.QuickMergeTop, Value: []string{"- IL"}},
+		{Name: "Exclude Poland (PL)", Action: config.QuickMergeTop, Value: []string{"- PL"}},
+		{Name: "Exclude United Kingdom (GB)", Action: config.QuickMergeTop, Value: []string{"- GB"}},
+		{Name: "Exclude United States of America (US)", Action: config.QuickMergeTop, Value: []string{"- US"}},
+	}
+
+	// SPNRulesVerdictNames defines the verdicts names to be used for SPN Rules.
+	SPNRulesVerdictNames = map[string]string{
+		"-": "Exclude", // Default.
+		"+": "Allow",
+	}
```

Comment:
I feel like the verdicts are not very clear on what they mean. What happens in `Exclude`? Is the app excluded from using the SPN for that domain?